frontend/debug.log

```diff
@@ -0,0 +1,2 @@
+[1214/075814.418:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
please remove this file and `package-lock.json` file from this commit as it's not required.